automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/AclEditor.scala

```diff
@@ -41,6 +41,16 @@ class AclEditor(implicit webDriver: WebDriver) extends OKCancelModal {
     roleDropdown.select(accessLevel.toString)
   }
 
+  def updateAccess(accessLevel: String): Unit = {
+    roleDropdown.select(accessLevel)
+  }
+
+  def removeCanCompute = canComputeBox.ensureUnchecked()
+
+  def canShareChecked: Boolean = canShareBox.isChecked
+
+  def canShareEnabled: Boolean = canShareBox.isEnabled
```

Comment:
If there are so many passthroughs like this, I wonder if the components should just be public and be accessed directly.